## Statystyka matematyczna I {-}

- Praca domowa 5

- Uladzislau Darhevich 108519

### Zadanie 1 {-}

$$X_1, X_2, ... X_n \sim  \text{Beta}(\theta, 1)$$



Wyznaczyć przedziały ufności dla $\theta$, jeżeli $n=$ 12, oraz $1 - \alpha = 0.98$

Kandydat na $U$ będzie: 

$$\sum_{i=1}^{n}(-\text{ln} X_i)$$

$$U = \theta\sum_{i=1}^{n}(-\text{ln} X_i)\Longrightarrow U\sim Beta(n,1)$$

$$a = F_{Beta(0,1)}^{-1}\bigg(\frac{\alpha}{2}\bigg)$$

$$b = F_{Beta(0,1)}^{-1}\bigg(1 - \frac{\alpha}{2}\bigg)$$

In [12]:
from scipy.stats import beta

alpha = 0.02
n = 12
a = beta.ppf(alpha/2, n, 1)
b = beta.ppf(1-alpha/2, n, 1)
print(f"a = {round(a, 3)}, b =  {round(b, 3)}")

a = 0.681, b =  0.999


$$\frac{0.681}{\sum_{i=1}^{n}-\text{ln}X_i}\le \theta\le \frac{0.999}{\sum_{i=1}^{n}-\text{ln}X_i}$$